decentraland/constants/Colors.js

```diff
@@ -13,7 +13,7 @@ export default {
   noticeText: "#fff",
   backgroundColor: "#fff",
   textColor: "gray",
-  headerBackgroundColor: "gray",
+  headerBackgroundColor: "#fff",
   headerTintColor: "#fff",
```

Comment:
We could even DRY up these color constants by having `const WHITE = #FFF`

Reply:
Done. And I've also removed unused colors.

Reply:
Thanks! Resolving